src/sets.jl

```diff
@@ -245,10 +245,26 @@ struct EqualTo{T<:Number} <: AbstractScalarSet
     value::T
 end
 
+"""
+    Parameter{T<:Number}(value::T)
+
+The set containing the single point ``x \\in \\mathbb{R}`` where ``x`` is given
+by `value`.
+
+This set is conceptually similar to the [`EqualTo`](@ref) set, except that a
+variable constrained to the `Parameter` set cannot have other constraints added
+to it, and the `Parameter` set can never be deleted. Thus, solvers are free to
```

Comment:
Should it always be used with `add_constrained_variable`? so that it would be ok to "delete" de variable, which will take the set with it...

Reply:
I guess the answer is "it depends." It'd be fine to add the variable then the constraint in `MOI.Utilities.Model`, but if it's a solver (or POI), it might not like getting the constraint after it has already created a variable.

So yes, you should probably always use `add_constrained_variable`, which also gets around the deletion issue, because you can't delete the constraint of a constrained variable.